package.json

```diff
@@ -33,13 +31,16 @@
   },
   "homepage": "https://github.com/CrowdStrike/eslint-config-crowdstrike#readme",
   "dependencies": {
-    "@eslint/js": "^9.0.0"
+    "@eslint/js": "^9.10.0",
```

Comment:
Open question is if we should keep this as a single bundle and assume the consuming project will treeshake the dependencies on build, or if we should have individual `package.json`'s for each ruleset, and be a true monorepo